lib/utils/mathMLTags.mjs

```diff
@@ -2,12 +2,11 @@ import { createRequire } from 'node:module';
 
 const require = createRequire(import.meta.url);
 
-// NOTE: mathml-tag-names v3 is a pure ESM package,
+// TODO: mathml-tag-names v3 is a pure ESM package,
 // so we cannot update it while supporting both ESM and CommonJS.
 //
 // In addition, mathml-tag-names v2 provides only a JSON file,
 // so ESM cannot import it (raises the "ERR_IMPORT_ASSERTION_TYPE_MISSING" error).
-// @ts-expect-error -- mathml-tag-names v2 doesn't have type declarations.
-const mathMLTags = require('mathml-tag-names');
+const mathMLTags = require('mathml-tag-names/index.json');
 
 export default mathMLTags;
```

Comment:
minor

Can you add
```js
/** @type {string[]} */
```
?

Reply:
Yes, this type is correctly inferred as `string[]`. However, specifying a type annotation may reduce type inference cost by `tsc` and help us understand the type more efficiently.

So I've added a type annotation via 6760c2cef.